portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,86 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.gson.Gson;
+import java.util.*;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+@WebServlet("/add-comment")
 public class DataServlet extends HttpServlet {
+  //This is a class to store a comment object.  
+  private class Comment {
+    String message, name;
+    public Comment(String message, String name){
+      this.message = message;
+      this.name = name;
+    }
```

Comment:
Add getters for the name and message fields. How would you read them? Using C++ structure type of reading as `obj.message` is not recommended. Always use setters and getters in Java.